### PR TITLE
product page

### PR DESCRIPTION
**Pull request by rrtk857**
## Overview

1. This PR fixes or fixes part of #9 
2. This PR does the following: This PR contain product page design with the use of API, HTML, CSS, JavaScript. the Product page has information about product and images, and buttons to buy it or add to cart. 
Screenshot (36)
Screenshot (37)
 

## Essential Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
- [x] The PR is made from a branch that's **not** called "main/master".

## PR Pointers

- If you need a review or an answer to a question, and don't have permissions to assign people, **leave a comment** like the following: "{{Question/comment}} @{{reviewer_username}} PTAL".
- Never force push. If you do, your PR will be closed.

### CODE REVIEW

**Comment by arminpatel** [comment]
@rrtk857 in case you wouldn't be updating your PR to use tailwind css, create a issue to fix this

**Comment by rrtk857** [comment]
@arminpatel, I will try to update it.

**Review by arminpatel — COMMENTED** [review]
@rrtk857  , great job so far. Just wanted to share a couple of suggestions with you:

- When you're contributing to a project, it's always helpful to go through the [CONTRIBUTING.md](https://github.com/bsoc-bitbyte/GetIt/blob/main/CONTRIBUTING.md#standard-commit-message-) file. It contains some conventions and guidelines to follow. It's a good idea to familiarize yourself with those before you start contributing.

- I noticed that your commit message style doesn't quite match what's mentioned in the CONTRIBUTING.md file for this project. No worries, though! It's completely fine for your first time contributing. Just remember to keep this in mind for your future contributions.

- Also, don't forget to update the PR title to match the recommended format, similar to the commit messages. Consistency in both commit messages and PR titles helps keep things organized and clear within the project.

File: pages/products/_id.vue [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+                    <button id="cart"><img src="https://cdn-icons-png.flaticon.com/512/711/711895.png">
+                        <p>Add To Cart</p>
+                    </button>
+                    <button id="buy"><img src="https://icon-library.com/images/current-icon/current-icon-1.jpg">
+                        <p>Buy Now</p>
+                    </button>
+                </div>
+            </div>
+        </div>
+        <div i="image_row">
```
Comment (thejaysharma) [inline pages/products/_id.vue:48]: hey @rrtk857, You can implement arrows function to show more product images Or you can make issue for the same. For reference :- https://www.youtube.com/watch?v=Y36QpYcnbQY&ab_channel=DennisIvy

File: pages/products/_id.vue [hunk 2]
```diff
@@ -0,0 +1,355 @@
+<template>
+    <div id="body">
+        <div id="upper">
+            <div id="circles">
```
Comment (thejaysharma) [inline pages/products/_id.vue:4]: the colour that the user is presently viewing should be active.

**Review by thejaysharma — CHANGES_REQUESTED** [review]
Hey @rrtk857, good job! Here are a few minor adjustments you may make.

**Comment by Priyansh61** [comment]
@JawHawk @tushitshukla03 PTAL!

**Comment by Priyansh61** [comment]
Hey @rrtk857 Please address the above comments. If you need any help do let us know :)

**Comment by rrtk857** [comment]
@arminpatel @thejaysharma @Priyansh61, Thanks for all the suggestions, I'll update my PR accordingly.
I have few doubts regarding tailwind CSS, I have installed it according to the given instructions, but I am bit confused in Step 2,3.
I am unable to understand where I have to include these files, so I have included it in the page folder only and it is working also.
So, I am attaching the screenshots below, please check it once that it is right, or I have to do any other changes.
Screenshot (41)
Screenshot (42)
Screenshot (43)
Screenshot (44)
Screenshot (45)

**Comment by thejaysharma** [comment]
@rrtk857 Tailwindcss has already been set up for your convenience. All you have to do is include it into your code.

**Comment by rrtk857** [comment]
I have made the changes and also used tailwind CSS.

https://github.com/bsoc-bitbyte/GetIt/assets/129523259/a6333827-2b9b-4d2b-88a1-fd5943f38115

**Comment by thejaysharma** [comment]
Push your changes so that we can review and merge it.

**Comment by rrtk857** [comment]
I have made some more changes. Please review it.

**Review by prernadabi23 — APPROVED** [review]

**Comment by rrtk857** [comment]
I have made all the required changes. Please review it.

https://github.com/bsoc-bitbyte/GetIt/assets/129523259/f4c9e02c-bb08-4257-8a98-ee3332f20690

**Review by Priyansh61 — APPROVED** [review]
LGTM :+1:

**Comment by rrtk857** [comment]
@Priyansh61, Difficulty of PR?

**Comment by Priyansh61** [comment]
@thejaysharma @prernadabi23 @tushitshukla03 label this PR plss?

**Pull request merged**